unikernel.ml

```diff
@@ -267,17 +271,21 @@ struct
           (* TODO(dinosaure): should report ICMP error message to src. *)
           ingest_private t end
 
-  let openvpn_configuration disk =
-    let* contents = KV.get disk (Mirage_kv.Key.v "/config.ovpn") in
+  let openvpn_configuration disk config_fn =
+    let* contents = KV.get disk (Mirage_kv.Key.v config_fn) in
     match contents with
-    | Error _ -> Fmt.failwith "No OpenVPN configuration found"
+    | Error _ ->
+      Logs.err(fun m -> m "Expected configuration file '%s' is absent in the root volume" config_fn);
+      Logs.err(fun m -> m "Try to run in dom0: qvm-volume import mirage-vpn:root vpn.tar");
+      Logs.err(fun m -> m "    with the tarball conatining the requested configuration file");
```

Comment:
I would do this in one log message as semantically it's one message (or maybe two).

Reply:
Thank you @reynir for your feedback, I hope [8149476](https://github.com/robur-coop/qubes-miragevpn/pull/4/commits/8149476a3bc218135af17ae12df8b3e2ad981f93) is what you had in mind :)
I don't know what I've done wrong, I can't see your avatar on the commit in the github GUI :(

Reply:
I think maybe you need a blank line before the `co-authored-by:` lines. Otherwise I have no idea :-)

Reply:
Yes you're right :)